src/plugins/tagfilter/tagfilter-fr.hbs

```diff
@@ -1000,6 +1001,254 @@
 &lt;/section></code></pre>
 </details>
 
+
+<section id="filteredFinder" class="wb-tagfilter provisional wb-filter" data-wb-tagfilter='{ "live": false }' data-wb-filter='{
+		"selector": "section",
+		"section": ".wb-tagfilter-items",
+		"uiTemplate": "#searchFilter2",
+		"live": false
+	}'>
+	<h2>Exemple 4 - Chercheur de types "ou" mis-à-jour avec bouton</h2>
+
+	<div class="row">
+	<search id="searchFilter2" class="col-xs-12 col-md-3 pull-right">
+		<h3>Filtres</h3>
+		<div>
+			<p class="wb-fltr-info mrgn-tp-0"><span data-nbitem></span> résultats sur <span data-total></span></p>
+		</div>
+
+		<form method="get">
+			<div class="form-group">
+				<label class="text-muted" for="customSearchUI2">Recherche par mor-clé</label>
+				<input type="search" class="form-control full-width" id="customSearchUI2">
+			</div>
+
+			<div class="form-group">
+			<fieldset>
+				<legend>Audience</legend>
+
+				<ul class="list-unstyled">
+					<li class="radio"><label><input type="checkbox" name="audience" value="student" class="wb-tagfilter-ctrl" data-wb-tagfilter='{"group": "orConditionGroup"}' /> Étudiant</label></li>
+					<li class="radio"><label><input type="checkbox" name="audience" value="worker" class="wb-tagfilter-ctrl" data-wb-tagfilter='{"group": "orConditionGroup"}' /> Travailleur</label></li>
+					<li class="radio"><label><input type="checkbox" name="audience" value="senior" class="wb-tagfilter-ctrl" data-wb-tagfilter='{"group": "orConditionGroup"}' /> Aîné</label></li>
```

Comment:
The bootlint task is freaking out because of the "radio" class being used with checkbox inputs (see [error E018](https://github.com/twbs/bootlint/wiki/E018)). So let's switch over to the "checkbox" class.

Btw EN isn't affected (it's already using the "checkbox" class).

```suggestion
					<li class="checkbox"><label><input type="checkbox" name="audience" value="student" class="wb-tagfilter-ctrl" data-wb-tagfilter='{"group": "orConditionGroup"}' /> Étudiant</label></li>
					<li class="checkbox"><label><input type="checkbox" name="audience" value="worker" class="wb-tagfilter-ctrl" data-wb-tagfilter='{"group": "orConditionGroup"}' /> Travailleur</label></li>
					<li class="checkbox"><label><input type="checkbox" name="audience" value="senior" class="wb-tagfilter-ctrl" data-wb-tagfilter='{"group": "orConditionGroup"}' /> Aîné</label></li>
```